src/main/java/uk/gov/justice/digital/hmpps/whereabouts/listeners/SqsOffenderEventListener.kt

```diff
@@ -48,16 +39,6 @@ class SqsOffenderEventListener(
             bookingIds,
           )
         }
```

Comment:
Did Jon B say to leave the prisoner deletion stuff for GDPR? I can't see it being used now, but not 100%

Reply:
I haven't spoken to Jon.